NOTICE

```diff
@@ -10,14 +10,44 @@ may not be used or distributed or accessed in any manner other than directly
 via Code.org's own web site, mobile applications, or social media channels,
 as-is without  modification or embedding in another format.
```

Comment:
Is there a reason the verbage for our proprietary materials is different than the verbage for third-party proprietary materials?